rest-api/src/main/java/com/cloudbees/workflow/rest/external/FlowNodeExt.java

```diff
@@ -167,10 +167,10 @@ public void addBasicNodeData(FlowNode node, String execNodeName, ExecDuration du
         setId(node.getId());
         setName(node.getDisplayName());
         setExecNode(execNodeName);
-        if (Stapler.getCurrentRequest() != null) {
+        if (get_links() == null) {
```

Comment:
get_links( ) will be null in most use cases: since we're re-initializing, is there any reason for this check?